app/controllers/pages_visited_controller.rb

```diff
@@ -0,0 +1,7 @@
+class PagesVisitedController < ApplicationController
+  def show
+    @phrase = Phrase.find(params[:id])
+    @unique_visitors_by_page = Page.find_by_sql('select pages.*, count(distinct(page_visits.visit_id)) as unique_visitors from pages join page_visits on page_visits.page_id = pages.id group by pages.id order by unique_visitors desc')
+    @@unique_visitors_by_page = []
```

Comment:
Is this class variable meant to be here?

Reply:
No, thanks 👍